### ISSUE TITLE
Fix keyboard navigation for the Accordion component

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#2130]
# Description

When using keyboard navigation on the [Accordion component](http://design.if-me.org/?path=/docs/components-accordion--regular-size-and-regular-style) ([code](https://github.com/ifmeorg/ifme/blob/main/client/app/components/Accordion/index.jsx)), the accordion is open and closed using the `TAB` key which is an incorrect interaction. 

It should only be toggled using the `ENTER` key.

The component is used in various places in the app including `/moments/new` and `/strategies/new`.

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!

**Comment by Syed-Ansar** [comment]
Hey @julianguyen I would like to work on this issue.

**Comment by julianguyen** [comment]
Hey @Syed-Ansar, I sent you an invitation to our org, so you can assign yourself to the issue. Thanks!

**Comment by Omkar0803** [comment]
I would like to contribute on this issue.

**Comment by MuhammadAakash** [comment]
I think no work is done on this issue till now.
Can I start working on the issue @julianguyen

**Comment by MuhammadAakash** [comment]
@Syed-Ansar are you still working on this issue?

**Comment by julianguyen** [comment]
@MuhammadAakash I can assign you to it since it hasn't been picked up in a couple of weeks!

**Comment by MuhammadAakash** [comment]
@julianguyen 
Sure i will start working on it.

**Comment by MuhammadAakash** [comment]
Hi @julianguyen 
I have just added a PR for this issue please review https://github.com/ifmeorg/ifme/pull/2198
